src/server/protocol_client.cc

```
@@ -52,21 +52,20 @@ using namespace facade;
 using absl::GetFlag;
 using absl::StrCat;
 
-namespace {
-
-error_code Recv(FiberSocketBase* input, base::IoBuf* dest) {
+error_code ProtocolClient::Recv(FiberSocketBase* input, base::IoBuf* dest) {
   auto buf = dest->AppendBuffer();
   io::Result<size_t> exp_size = input->Recv(buf);
-  if (!exp_size)
+  if (!exp_size) {
+    LOG(ERROR) << "Socket error " << exp_size.error();
```

I suggest log(warning). otherwise control plane suppress this error I think

---

Fixed.